.github/workflows/build.yml

```diff
@@ -103,6 +103,25 @@ jobs:
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.name }}
     steps:
+
+    # sccache's handling of the /fp:fast MSVC compiler option is broken, so use our fork with the fix.
+    # https://github.com/mozilla/sccache/issues/950
+    - name: "[Windows] Set up cargo cache"
+      if: runner.os == 'Windows'
+      uses: actions/cache@v2
+      id: sccache-build-cache
+      with:
+        path: C:\Users\runneradmin\.cargo
+        # hash of commit to build
+        key: e12434e4db24ccbe52b83fb6e37afcb94fc8d37b
+
+    # This needs to be done first because something later messes with $PATH in a way that breaks cargo
+    # by causing it to find link.exe from Git for Windows instead of MSVC.
+    - name: "[Windows] Build fixed sccache"
+      shell: bash
+      if: runner.os == 'Windows' && steps.sccache-build-cache.outputs.cache-hit != 'true'
+      run: cargo install --git https://github.com/Be-ing/sccache.git --branch fix_msvc_fp
```

Comment:
Please add a comment why this patched version is needed and a TODO that explains how to replace it later (including a link to the PR).

Reply:
The comment is on the above step. I can move it here.

Reply:
I am torn whether to move the comment down here or leave it where it is. What do you think?

Reply:
> But including the final version of the cargo install command as a comment would still be helpful.

This is IMO obvious from normal usage of cargo and the sccache documentation, but I added a comment anyway.